__tests__/app.test.js

```diff
@@ -98,6 +94,49 @@ describe("/api/review/review_id", () => {
         expect(res.body.msg).toBe("bad request");
       });
   });
+
+  test("PATCH - 200: when correct info passed, update votes field, respond with updated review object", () => {
+    const updatedInfo = { inc_votes: 2 };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(updatedInfo)
+      .expect(200)
+      .then((res) => {
+        expect(res.body.review).toMatchObject({
+          title: "Agricola",
+          designer: "Uwe Rosenberg",
+          owner: "mallionaire",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          review_body: "Farmyard fun!",
+          category: "euro game",
+          created_at: expect.any(String),
+          votes: 3,
+        });
+      });
+  });
+  test(" PATCH - 400  when no info passed, respond with 400", () => {
+    const updatedInfo = {}
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(updatedInfo)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe('bad request')
+      })
+  });
+  test(" PATCH - 400  when incorrect info passed, respond with 400", () => {
+    const updatedInfo = {
+      inc_votes: 'test'
+    }
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(updatedInfo)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe('bad request')
+      })
+  });
```

Comment:
Needs a couple of tests relating to the articleID itself - when its not a number and when the article is simply not found (eg id of 9999)